src/sas/sasgui/perspectives/fitting/models.py

```diff
@@ -153,7 +153,7 @@ def compile_file(dir):
     """
     try:
         import compileall
-        compileall.compile_dir(dir=dir, ddir=dir, force=1,
+        compileall.compile_dir(dir=dir, ddir=dir, force=0,
                                quiet=report_problem)
```

Comment:
Agree. This was completely unintended (an old commit to models.py that I don't know how it ended here). Now went back to the original version of fitting.py in order to clear the plot (show zeros) when the model does not return any finite value. With this the only change is that the number of finite values returned by the model is stored and used at the end of the _complete1d and _complete2d functions to show the appropriate messages in the screen.